app/src/main/java/org/fossasia/susi/ai/skills/feedback/adapters/recycleradapters/AllReviewsAdapter.kt

```diff
@@ -38,7 +41,8 @@ class AllReviewsAdapter(val context: Context, val feedbackList: List<Feedback>?)
             if (feedbackList[position] != null) {
                 if (feedbackList[position].email != null &&
                         !TextUtils.isEmpty(feedbackList[position].email)) {
-                    holder.initials.text = feedbackList[position].email?.substring(0, 2)
+                    val imageUrl: String = Utils.getGravatarUrl(Utils.toMd5Hash(feedbackList.get(position).email))
```

Comment:
Directly send email in the util method, it will implicitly call the toMd5Hash method